CODEOWNERS

```diff
@@ -1,2 +1,11 @@
-CODEOWNERS @guidanoli
+# ABOUT CODE OWNERS
+# THIS FILE FOLLOWS https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+CODEOWNERS @guidanoli @xdaniortega @ZzzzHui @pedroargento
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @@guidanoli @xdaniortega @ZzzzHui @pedroargento will be requested
+# for review when someone opens a pull request.
+* @guidanoli @xdaniortega @ZzzzHui @pedroargento
+
```

Comment:
I would also remove these lines.
To put everyone as code owner of everything is the same as not doing so.

Reply:
Don't really agree here, we can find in a near feature, possible external contributors. I find reasonable to make all the Rollup Ref Unit members owners of the code, and then, make it mandatory for at least one of us to approve changes.

Reply:
This is already the current behavior, since only the Rollups Ref Unit has write access to the repository and, therefore, can approve/merge PRs.

Reply:
Not really the exact same behavior. For PRs, right now, there is the only one profile that shows up mandatory to approve it, I propose to make it fair and equal to all of the members.

Also the proposal fits perfectly the purpose of this [file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):
`You can use a CODEOWNERS file to define individuals or teams that are responsible for code in a repository.`

Reply:
> For PRs, right now, there is the only one profile that shows up mandatory to approve it, I propose to make it fair and equal to all of the members.

Because this PR proposes to change the `CODEOWNERS` file, of which I am the only owner for now, I am the only one mandatory for this PR. I do agree, however, to allow everyone from the Rollups Ref Unit to be able to change it. Still, it's worth noting that, for PRs that do not change `CODEOWNERS`, there is no mandatory reviewer.

Adding everyone in the team as a code owner of all the repository wouldn't change the requirements for a PR to be merged, since, by the repository configuration, there must be at least one approval in order for the PR to be merged, and currently only the members of the Rollups Ref Unit are capable of approving PRs.